### PR TITLE
Sequence and most common character.

### PR DESCRIPTION
**Pull request by AlexanderVladimirov9090**
Done

### CODE REVIEW

File: sentence/src/Text.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  /**
+   * Traverse character frequency to return the most common character in given text.
+   * @return most common character.
+   */
+  public Character mostCommonCharacter() {
+    Integer maxValue = Collections.max(charactersFrequency().values());
+    Map<Character, Integer> charactersFrequency = charactersFrequency();
+    for (Character eachCharacter : charactersFrequency.keySet()) {
+
+      if (charactersFrequency().get(eachCharacter).equals(maxValue)) {
```
Comment (mgenov) [inline sentence/src/Text.java]: On each iteration frequency is counted which is not a good idea.

File: sentence/src/Text.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  /**
+   * Traverse character frequency to return the most common character in given text.
+   * @return most common character.
+   */
+  public Character mostCommonCharacter() {
+    Integer maxValue = Collections.max(charactersFrequency().values());
+    Map<Character, Integer> charactersFrequency = charactersFrequency();
+    for (Character eachCharacter : charactersFrequency.keySet()) {
+
+      if (charactersFrequency.get(eachCharacter).equals(maxValue)) {
```
Comment (AlexanderVladimirov9090) [inline sentence/src/Text.java]: You want to say like this?

**Comment by mgenov** [comment]
LGTM !

**Comment by AlexanderVladimirov9090** [comment]
I think it is done.

**Pull request merged**